containers/runner/README.md

```diff
@@ -137,3 +137,16 @@ the graphical console for a virtual machine with a command:
 
 To change of the boot options for the tests (for example to add `inst.text`), please use
 `--run-args="--env KSTEST_EXTRA_BOOTOPTS=inst.text"` parameter for the `launch` script.
+
+## Running test manually from the container
+
+Sometimes it could be helpful to run the test in the container manually to be able to inspect the
+container after the run. To do that follow this guide:
+
+Run `containers/runner/launch` with `-c` argument in additional to other arguments. This will
+open shell of the container where you can do what you want to before starting the test. To run
+the test(s) execute
+
+    /kickstart-tests/containers/runner/run-kstest
+
+to start the test execution.
```

Comment:
If you want some writing suggestions...

```suggestion
container after the run. To do that:

1. Run `containers/runner/launch` with `-c` argument in addition to other arguments. This will
open shell of the container where you can do what you want to before starting the test.
2. Start the test(s) execution with `/kickstart-tests/containers/runner/run-kstest`
```

Reply:
Thanks for a great suggestion.